public/src/components/script/script.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+
+export default class ScriptPage extends Component {
+  constructor(props) {
+    super(props);
+    this.renderQuestions = this.renderQuestions.bind(this);
+  }
+
+  componentDidMount() {
+    const { script_id } = this.props;
+    this.props.fetchScript(script_id);
+    this.props.fetchScriptQuestions(script_id);
+  }
+
+  renderQuestions() {
```

Comment:
This could be a component instead of a method in this component. Perfect use case for a stateless component.